README.md

```diff
@@ -128,8 +128,10 @@ The following haml snippet is an example of the code to include on your html pag
         %dt.label{  :data => { value: "Rating" } }
     %dl.quick-filters
       %dl.author-block
+        %dt.label{ data: "Author Block Label"}
```

Comment:
should this be the same configuration as the others? ie
{ :data => { value: "Author Block Label" }}